filter_list/list_filter.py

```diff
@@ -1,7 +1,10 @@
-from list_filter_example.filter_list import lists_class_filter
+from unittest import result
```

Comment:
why are you importing result if you are overwriting result at line number 8.
I think this import is not used.